### PR TITLE
[WIP] Use namespaced classes of PHPUnit

### PR DESCRIPTION
**Pull request by carusogabriel**
## Description

As we minimum require PHP 7.1, we can require this PHPUnit version. The PHPUnit 7 version requires more work.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]
This will break all apps ...

- Apps use core's phpunit to run their tests
- Apps are usually written to work on ooc10+
- This requires php5.6

As long as oc10 is maintained we cannot merged this

But: feel free to migrate the code to work with phpunit 6 - older phpunit versions have a compatibility layer in place. THX

**Comment by carusogabriel** [comment]
@DeepDiver1975 Gotcha. Following @phil-davis's suggestion as well, I've update this patch.

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]
AFAIK there are cs-fixer rules for PHPUnit as well ... please have a look ... THX

File: apps/files/tests/Controller/ViewControllerTest.php [hunk 1]
```diff
@@ -432,7 +432,7 @@ public function testShowFileRouteWithInvalidFileId($useShowFile) {
 			->will($this->returnValue([]));
 
 		if ($useShowFile) {
-			$this->setExpectedException('OCP\Files\NotFoundException');
+			$this->expectException('OCP\Files\NotFoundException');
```
[the remark this answers is not in the document]
Comment (phil-davis) [reply apps/files/tests/Controller/ViewControllerTest.php]: @carusogabriel "I will later create"  or "I would hate to create" or?
The auto-spell-checker in my brain is confused :)

[the remark this answers is not in the document]
Comment (carusogabriel) [reply apps/files/tests/Controller/ViewControllerTest.php]: @phil-davis Sorry, misspelling :man_facepalming:, edited

File: tests/acceptance/features/bootstrap/CapabilitiesContext.php [hunk 2]
```diff
@@ -62,7 +62,7 @@ public function checkCapabilitiesResponse(TableNode $formData) {
 			if ($row['value'] === "%versionstring%") {
 				$row['value'] = $this->featureContext->getVersionStringFromStatus();
 			}
-			PHPUnit_Framework_Assert::assertEquals(
+			PHPUnit\Framework\Assert::assertEquals(
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/CapabilitiesContext.php]: no ``use`` added for this file?

File: tests/acceptance/features/bootstrap/CommandLine.php [hunk 3]
```diff
@@ -435,7 +435,7 @@ public function theCommandOutputContainsTheText($text) {
 		// end of the captured string, so trim them.
 		$text = \trim($text, $text[0]);
 		$lines = $this->findLines($this->lastStdOut, $text);
-		PHPUnit_Framework_Assert::assertGreaterThanOrEqual(
+		PHPUnit\Framework\Assert::assertGreaterThanOrEqual(
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/CommandLine.php]: no ``use`` added for this file?

File: tests/acceptance/features/bootstrap/Comments.php [hunk 4]
```diff
@@ -118,7 +118,7 @@ public function checkComments($user, $path, $expectedElements) {
 						break;
 					}
 				}
-				PHPUnit_Framework_Assert::assertTrue(
+				PHPUnit\Framework\Assert::assertTrue(
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/Comments.php]: no ``use`` added for this file?

File: tests/acceptance/features/bootstrap/EmailContext.php [hunk 5]
```diff
@@ -60,7 +60,7 @@ public function assertThatEmailContains($address, PyStringNode $content) {
 		$expectedContent = $this->featureContext->replaceProductName(
 			$expectedContent
 		);
-		PHPUnit_Framework_Assert::assertContains(
+		PHPUnit\Framework\Assert::assertContains(
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/EmailContext.php]: no ``use`` added for this file?

File: tests/acceptance/features/bootstrap/Logging.php [hunk 6]
```diff
@@ -70,19 +70,19 @@ public function theLastLinesOfTheLogFileShouldContainEntriesWithTheseAttributes(
 					= $this->featureContext->substituteInLineCodes(
 						$expectedLogEntry[$attribute]
 					);
-				PHPUnit_Framework_Assert::assertArrayHasKey(
+				PHPUnit\Framework\Assert::assertArrayHasKey(
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/Logging.php]: no ``use`` added for this file?

File: tests/acceptance/features/bootstrap/SearchContext.php [hunk 7]
```diff
@@ -98,22 +98,22 @@ public function fileInTheSearchResultShouldContainProperties(
 		$fileResult = $this->featureContext->findFileFromPropfindResponse(
 			$user, $file
 		);
-		PHPUnit_Framework_Assert::assertNotFalse(
+		PHPUnit\Framework\Assert::assertNotFalse(
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/SearchContext.php]: no ``use`` added for this file?

File: tests/acceptance/features/bootstrap/ShareesContext.php [hunk 8]
```diff
@@ -91,7 +91,7 @@ public function theShareesReturnedShouldBe($shareeType, $shareesList) {
 		$respondedArray = $this->getArrayOfShareesResponded(
 			$this->featureContext->getResponse(), $shareeType
 		);
-		PHPUnit_Framework_Assert::assertEquals($sharees, $respondedArray);
+		PHPUnit\Framework\Assert::assertEquals($sharees, $respondedArray);
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/ShareesContext.php]: no ``use`` added for this file?

File: tests/acceptance/features/bootstrap/WebUIAdminGeneralSettingsContext.php [hunk 9]
```diff
@@ -208,7 +208,7 @@ public function before(BeforeScenarioScope $scope) {
 	public function theVersionOfOwncloudInstallationShouldBeDisplayedOnTheAdminGeneralSettingsPage() {
 		$actualVersion = $this->adminGeneralSettingsPage->getOwncloudVersion();
 		$expectedVersion = $this->featureContext->getSystemConfigValue('version');
-		PHPUnit_Framework_Assert::assertEquals(\trim($expectedVersion), $actualVersion);
+		PHPUnit\Framework\Assert::assertEquals(\trim($expectedVersion), $actualVersion);
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/WebUIAdminGeneralSettingsContext.php]: no ``use`` added for this file?

File: tests/acceptance/features/bootstrap/WebUILoginContext.php [hunk 10]
```diff
@@ -404,7 +404,7 @@ public function theUserRequestsThePasswordResetLinkUsingTheWebui() {
 	public function thisMessageShouldBeDisplayed(PyStringNode $string) {
 		$expectedString = $string->getRaw();
 		$passwordRecoveryMessage = $this->loginPage->getLostPasswordMessage();
-		PHPUnit_Framework_Assert::assertEquals(
+		PHPUnit\Framework\Assert::assertEquals(
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/WebUILoginContext.php]: no ``use`` added for this file?

File: tests/acceptance/features/bootstrap/WebUINotificationsContext.php [hunk 11]
```diff
@@ -62,7 +62,7 @@ public function assertNotificationsOnWebUI(
 	) {
 		$notificationsDialog = $this->openNotificationsDialog();
 		$notifications = $notificationsDialog->getAllNotifications();
-		PHPUnit_Framework_Assert::assertEquals(
+		PHPUnit\Framework\Assert::assertEquals(
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/WebUINotificationsContext.php]: no ``use`` added for this file?

File: tests/acceptance/features/bootstrap/WebUIPersonalGeneralSettingsContext.php [hunk 12]
```diff
@@ -176,7 +176,7 @@ public function theUserChangesTheEmailAddressToUsingTheWebUI($emailAddress) {
 	 * @return void
 	 */
 	public function theOwncloudVersionShouldBeDisplayedOnThePersonalGeneralSettingsPageInTheWebui() {
-		PHPUnit_Framework_Assert::assertTrue($this->personalGeneralSettingsPage->isVersionDisplayed());
+		PHPUnit\Framework\Assert::assertTrue($this->personalGeneralSettingsPage->isVersionDisplayed());
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/WebUIPersonalGeneralSettingsContext.php]: no ``use`` added for this file?

File: tests/acceptance/features/bootstrap/WebUIPersonalSecuritySettingsContext.php [hunk 13]
```diff
@@ -91,10 +91,10 @@ public function theAppShouldBeListedInTheAppPasswordsListOnTheWebUI() {
 		$appTr = $this->personalSecuritySettingsPage->getLinkedAppByName(
 			$this->appName
 		);
-		PHPUnit_Framework_Assert::assertNotEmpty($appTr);
+		PHPUnit\Framework\Assert::assertNotEmpty($appTr);
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/WebUIPersonalSecuritySettingsContext.php]: no ``use`` added for this file?

**Review by phil-davis — COMMENTED** [review]
Some files do not yet have the ``use`` statement added.

**Comment by carusogabriel** [comment]
Closing as I'm gonna work on updating the version of PHPUnit instead

**Pull request closed without merging**